### PR TITLE
[BUG] Remove silently ignored --pg-driver CLI option.

### PR DESCRIPTION
**Pull request by dargueta**
Version 1.0.2 removed the ability to specify your Postgres driver but didn't remove the ``--pg-driver`` CLI option. This resulted in a bug where passing ``--pg-driver`` to `pytest` wouldn't change the driver. Since there's no way to work around this, we have to remove the CLI option.

@wesleykendall

### CODE REVIEW

File: test_requirements.txt [hunk 1]
```diff
@@ -10,7 +10,8 @@ pytest>=3.0,<3.3; python_version<"3.4"
 coverage==4.4.2
 flake8==3.5.0
 psycopg2==2.7.3
-pylint==1.7.4
+pylint==1.8.1; python_version>'3.3'
+pylint==1.7.5; python_version=='3.3'
```
Comment (wesleykendall) [inline test_requirements.txt]: What's the reason for this change?

**Comment by wesleykendall** [comment]
@dargueta lgtm, just had a question about pylint

File: test_requirements.txt (hunk 1 above)
Comment (dargueta) [reply test_requirements.txt]: Python 1.8 dropped support for Python 3.3.

**Comment by wesleykendall** [comment]
@dargueta let me know and I'll merge if you have no other additions

**Comment by dargueta** [comment]
>  let me know

Uh... do you mean this? https://github.com/CloverHealth/pytest-pgsql/pull/3#discussion_r160490805

> I'll merge if you have no other additions

Nah, I'm done. Good to go on my end.

**Comment by wesleykendall** [comment]
@dargueta do you want me to do an api break so that it will be on 2.0? I am fine with keeping it on 1.0 since this feature wasn't being used by anyone yet. I'd kinda like to keep the api breaks only when we truly break the API and release a big change. Let me know what you think

**Comment by dargueta** [comment]
Er... I already put `api-break` in the first commit message. Will `pbr` ignore it if you change the merge commit message?

Anyway, I'll leave it up to you.

**Comment by wesleykendall** [comment]
@dargueta only the message in the squashed commit will be used by PBR. I'll ignore doing an API break for this
